specs/lightclient/beacon-chain.md

```diff
@@ -166,6 +242,92 @@ def get_sync_committee(state: BeaconState, epoch: Epoch) -> SyncCommittee:
     return SyncCommittee(pubkeys=pubkeys, pubkey_aggregates=aggregates)
 ```
 
+#### `get_base_reward`
+
+*Note*: The function `get_base_reward` is modified with the removal of `BASE_REWARDS_PER_EPOCH`.
+
+```python
+def get_base_reward(state: BeaconState, index: ValidatorIndex) -> Gwei:
```

Comment:
base reward to me sounds more like a baseline aka the minimal. I'd consider renaming this to `get_max_reward`

Reply:
But `get_base_reward`  does not have a baseline meaning in the current spec. The naming is consistent with the current behavior. Changing it may cause more confusion